Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.h

```diff
@@ -184,7 +184,7 @@ namespace Multiplayer
         double m_serverSendAccumulator = 0.0;
         float m_renderBlendFactor = 0.0f;
         float m_tickFactor = 0.0f;
-        bool m_spawnNetboundEntities = true;
+        bool m_spawnNetboundEntities = false;
```

Comment:
If you ctrl-G, escape, and ctrl-G again does this properly get reset to false?

Reply:
The editor is a client so it'll stay false. Escaping gamemode will turn the agent type from "client" back to "unintialized" though.